flow-decomposition/src/main/java/com/powsybl/flow_decomposition/rescaler/DecomposedFlowRescalerProportional.java

```diff
@@ -8,16 +8,15 @@
 package com.powsybl.flow_decomposition.rescaler;
 
 import com.powsybl.flow_decomposition.DecomposedFlow;
-import com.powsybl.flow_decomposition.DecomposedFlowBuilder;
-import com.powsybl.iidm.network.Country;
+import com.powsybl.iidm.network.Network;
 
 import java.util.Map;
 import java.util.stream.Collectors;
 
 /**
  * @author Caio Luke {@literal <caio.luke at artelys.com>}
  */
-public class DecomposedFlowRescalerProportional implements DecomposedFlowRescaler {
+public class DecomposedFlowRescalerProportional extends AbstractDecomposedRescaler {
```

Comment:
Maybe we should move `DEFAULT_MIN_FLOW_TOLERANCE` to `FlowDecompositionParameters` to reverse the dependency direction

Reply:
Which dependency direction?

Reply:
We should move DEFAULT_MIN_FLOW_TOLERANCE to FlowDecompositionParameters.

Thus the rescalers default tolerance will depend on the FlowDecompositionParameters

Reply:
Ah, I get it now! I must've been tired on friday, I didn't realize you were talking about the constant.
Yes, it makes total sense to move it there.
I just did it.